modules/aws-iam-policies/main.tf

```diff
@@ -23,10 +24,19 @@ data "aws_iam_policy_document" "emr_creator_policy" {
       "elasticmapreduce:TerminateJobFlows",
       "iam:PassRole"
     ]
-    resources = [
-      "arn:${var.arn_partition}:iam::${data.aws_caller_identity.current.account_id}:role/*",
-      "arn:${var.arn_partition}:elasticmapreduce:*:${data.aws_caller_identity.current.account_id}:cluster/*"
-    ]
+    resources = flatten([
```

Comment:
We could get rid of this flatten if we split the `"iam:PassRole"` into its own policy since I don't think the EMR actions need the iam resource.

Reply:
leaving as is unless you prefer otherwise